worldedit-bukkit/adapters/adapter-1_19_4/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R3/PaperweightPlatformAdapter.java

```diff
@@ -138,8 +138,15 @@ public final class PaperweightPlatformAdapter extends NMSAdapter {
             fieldTickingFluidCount.setAccessible(true);
             fieldTickingBlockCount = LevelChunkSection.class.getDeclaredField(Refraction.pickName("tickingBlockCount", "g"));
             fieldTickingBlockCount.setAccessible(true);
-            fieldNonEmptyBlockCount = LevelChunkSection.class.getDeclaredField(Refraction.pickName("nonEmptyBlockCount", "f"));
-            fieldNonEmptyBlockCount.setAccessible(true);
+            Field tmpFieldBiomes;
+            try {
+                // It seems to actually be biomes, but is apparently obfuscated to "j"
+                tmpFieldBiomes = LevelChunkSection.class.getDeclaredField("biomes");
+            } catch (NoSuchFieldException ignored) {
+                tmpFieldBiomes = LevelChunkSection.class.getDeclaredField("j");
+            }
+            fieldBiomes = tmpFieldBiomes;
```

Comment:
Whoops one more question, why isn't this using Refraction like the other fields?

Reply:
> // It seems to actually be biomes, but is apparently obfuscated to "j"

MiniMappingViewer said `j` but when actually using it, it was `biomes` for whatever reason

Reply:
If you tested that on paper, you probably encountered the remapped version - but paper should also successfully remap `j` to `biomes` then

Reply:
It works for me on 1.20.6

Reply:
I was on 1.20.4, I also didn't bother checking every version, just assumed it could be an issue there as well

Reply:
It also works on 1.20.4 for me, Refraction properly detects that it's running in a mojmapped environment

Reply:
I'll stick it back to normal and assume it was something off just once